sample/application/controller/UserController.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace sample\application\controller;
+
+use sample\data\dataSource\UserStorageDoctrineDataSource;
+use sample\data\mapper\UserEntityToUserMapper;
+use sample\data\mapper\UserToUserEntityMapper;
+use sample\data\repository\doctrine\operation\CreateUserOperation;
+use sample\domain\entity\User;
+use src\data\dataSource\query\IntegerIdQuery;
+use src\data\dataSource\query\Query;
+use src\data\repository\operation\Operation;
+use src\data\repository\RepositoryMapper;
+use src\data\repository\SingleDataSourceRepository;
+use src\domain\interactor\DeleteInteractor;
+use src\domain\interactor\GetAllInteractor;
+use src\domain\interactor\GetInteractor;
+use src\domain\interactor\PutInteractor;
+
+class UserController {
+
+    public function provideGetInteractor()
+    {
+        $dataSource = new UserStorageDoctrineDataSource();
+        $repository = new SingleDataSourceRepository(
+            $dataSource,
+            $dataSource,
+            $dataSource
+        );
+        $repositoryMapper = new RepositoryMapper(
+            $repository,
+            $repository,
+            $repository,
+            new UserToUserEntityMapper(),
+            new UserEntityToUserMapper()
+        );
+        return new GetInteractor($repositoryMapper);
+    }
+
+    /**
+     * Get user
+     *
+     * @param $userId
+     *
+     * @return array
```

Comment:
We don't return a collection of users?

Reply:
Yes, we spoke with Jose about that, and we wanted to discuss it a before so, for the first iteration, I left it like this.